WalletWasabi.Gui/CommandLine/Daemon.cs

```diff
@@ -78,7 +79,10 @@ internal async Task RunAsync(string walletName, bool mixAll, bool keepMixAlive)
 					return;
 				}
 
-				WalletService = await Global.CreateWalletServiceAsync(keyManager);
+				var walletManager = Global.WalletManager;
+				WalletService = await walletManager.AddWalletServiceAsync(keyManager);
```

Comment:
`Add` implies "I already have the object and I want to add it.

however this method is more like `Create + Add`.

Reply:
I tried that. Then I had problems with the RegTests where a valid scenario appeared. I need the WalletService itself, subscribe to some of its events then pass it for some other classes and just then start it. We will have more of this when starting to have real multi-wallet.

Reply:
That wasn't @danwalmsley's point. He simply suggested that the `CreateWalletServiceAsync` naming would be better. And I agree. Done.